Userland/Games/MasterWord/WordGame.cpp

```diff
@@ -82,10 +90,17 @@ void WordGame::keydown_event(GUI::KeyEvent& event)
         m_current_guess = m_current_guess.substring(0, m_current_guess.length() - 1);
         m_last_word_not_in_dictionary = false;
     }
-    // If enough letters and return pressed
-    else if (m_current_guess.length() == m_num_letters && event.key() == KeyCode::Key_Return) {
-        if (is_in_dictionary(m_current_guess)) {
+    // If return pressed
+    else if (event.key() == KeyCode::Key_Return) {
+        if (m_current_guess.length() < m_num_letters) {
+            show_message("Not enough letters"sv);
+        } else if (!is_in_dictionary(m_current_guess)) {
+            show_message("Not in dictionary"sv);
+            m_last_word_not_in_dictionary = true;
+        } else {
             m_last_word_not_in_dictionary = false;
+            show_message("go on"sv);
```

Comment:
Is this message just for debugging?
Since we show other visual changes I think just immediately clearing the message seems like a better choice here.

Reply:
Yep,  the message was for debugging. Forgot it there :)